tests/e2e/fixtures/statsFixtures.js

```diff
@@ -263,6 +262,64 @@ const seasonTwoMatches = [
   },
 ];
 
+const seasonFourMatches = [
+  ////////////
+  // Week 1 //
+  ////////////
+  // Player 1 week 1
+  {
+    player1: 'player1',
+    player2: 'player2',
+    winner: 'player2',
+    startTime: dayjs().subtract(1, 'day').valueOf(),
+    endTime: dayjs().subtract(1, 'day').valueOf(),
+  },
+  {
+    player1: 'player1',
+    player2: 'player3',
+    winner: 'player1',
+    startTime: dayjs().subtract(2, 'day').valueOf(),
+    endTime: dayjs().subtract(2, 'day').valueOf(),
+  },
+  {
+    player1: 'player1',
+    player2: 'player4',
+    winner: 'player4',
+    startTime: dayjs().add(1, 'week').valueOf(),
+    endTime: dayjs().add(1, 'week').valueOf(),
+  },
+  // Player 2 week 1
+  {
+    player1: 'player2',
+    player2: 'player3',
+    winner: 'player3',
+    startTime: dayjs().add(2, 'week').valueOf(),
+    endTime: dayjs().add(2, 'week').valueOf(),
+  },
+  {
+    player1: 'player2',
+    player2: 'player4',
+    winner: 'player2',
+    startTime: dayjs().add(3, 'week').valueOf(),
+    endTime: dayjs().add(3, 'week').valueOf(),
+  },
+  // Player 3 week 1
+  {
+    player1: 'player3',
+    player2: 'player4',
+    winner: 'player3',
+    startTime: dayjs().add(4, 'week').valueOf(),
+    endTime: dayjs().add(4, 'week').valueOf(),
+  },
+  {
+    player1: 'player3',
+    player2: 'player5',
+    winner: 'player3',
+    startTime: dayjs().add(5, 'week').valueOf(),
+    endTime: dayjs().add(5, 'week').valueOf(),
+  },
```

Comment:
We can probably remove these altogether since we don't actually need them for the tests
```suggestion
```